ml-stat/ml-stat-tests.ts

```diff
@@ -0,0 +1,102 @@
+/// <reference path="./index.d.ts" />

```

Comment:
Do you try with `./node_modules/.bin/tsc -p ./ml-stat/tsconfig.json` ?
If tsc find https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12364/files#diff-a0609d86dd1b8ba87000fb69b42f6392R17 this configurations, tsc don't raise any errors. I think.